core/templates/pages/exploration-player-page/layout-directives/progress-nav.directive.ts

```diff
@@ -20,7 +20,7 @@ import { Subscription } from 'rxjs';
 
 require(
   'pages/exploration-player-page/learner-experience/' +
-  'continue-button.directive.ts');
+  'continue-button.component.ts');
```

Comment:
I don't see this file in the PR.. Are some files missing here?

Reply:
I renamed the file _continue-button.directive.ts_ to _continue-button.component.ts_, after making the changes, and followed the suggested git procedure to push the branch given in [https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia](url), but I am not sure why this file is not appearing in the PR. Can you please suggest the fix.